core/app/models/concerns/spree/default_price.rb

```diff
@@ -16,9 +16,15 @@ def self.default_price_attributes
 
     # Returns `#prices` prioritized for being considered as default price
     #
-    # @return [ActiveRecord::Relation<Spree::Price>]
+    # @return [Array<Spree::Price>]
     def currently_valid_prices
-      prices.currently_valid
+      prices.sort_by do |price|
```

Comment:
If we don't use the `currently_valid` scope here... do we use it anywhere? I believe my stores rely on the behaviour of `currently_valid`, though we can obviously move things around if needed.

Reply:
We use it in the admin prices controller. We could probably get rid of it, but I would say that's outside of the scope of this PR :)